simple_ai_agents/newsletter_agent/.env.example

```
@@ -1,2 +1,43 @@
-NEBIUS_API_KEY="Your Nebius Api key"
-FIRECRAWL_API_KEY="Your Firecrawl API Key"
\ No newline at end of file
+﻿# =============================================================================
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)